CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -15,7 +15,8 @@ public class HarvestGateway : IHarvestDeveloperRetriever, ITimeSheetRetriever
         private readonly string _token;
         private readonly string _accountId;
         private readonly string _userAgent;
-        private readonly List<string> _listOfDeveloperRoles = new List<string>
+        private readonly string[] _listOfDeveloperRoles;
+        /*private readonly List<string> _listOfDeveloperRoles = new List<string>
```

Comment:
Don't need this anymore

Reply:
Thanks! 
Removed in https://github.com/chubberlisk/crypto_tech/pull/31/commits/f82406849e496ffa056ddc29e95e64c6c85c734a.